lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -16,22 +17,31 @@
             },
 
             showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+                var accountSettingsLink = HtmlUtils.joinHtml(
+                    HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML('<a href="{settings_url}">', {settings_url: this.options.accountSettingsPageUrl})
+                    ),
+                    gettext('Account Settings page.'),
+                    HtmlUtils.HTML('</a>')
+                );
                 if (this.profileIsPrivate) {
-                    this._super(interpolate_text(
-                        gettext("You must specify your birth year before you can share your full profile. To specify your birth year, go to the {account_settings_page_link}"),
-                        {'account_settings_page_link': accountSettingsLink}
-                    ));
+                    this._super(
+                        HtmlUtils.interpolateHtml(
```

Comment:
Oh, my mistake. I see that the link is being passed in, so there is HTML in line. Sorry for the confusion.